src/laundryScraper.py

```diff
@@ -115,21 +115,21 @@ def scrapeLaundry(event, context):
 """
 def writeDynamo(buildingList, machineDict):

```

Comment:
Pass the buildingTable and machineTable objects in as a parameter, created in the handler. Like I mentioned earlier in the review, it's a good idea to have a function just called 'handler' which just creates the Dynamo stuff and then calls another function which does all the real work.